sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -126,6 +127,7 @@ int main(int argc, char **argv) {
         << std::endl
         << std::endl;
   }
+#endif
```

Comment:
I don't think we build `sycl-ls` with `-fpreview-breaking-changes` ever, so there is really no point in masking this out now, other than maybe to remind us to remove it in the future.